gitops/snapshot.go

```diff
@@ -194,51 +194,51 @@ func PrepareToRegisterIntegrationPipelineRun(snapshot *applicationapiv1alpha1.Sn
 	go metrics.RegisterNewIntegrationPipelineRun(snapshot.GetCreationTimestamp(), pipelineRunStartTime)
 }
 
-// SetSnapshotIntegrationStatusAsFinished sets the HACBS integration status condition for the Snapshot to Finished.
+// SetSnapshotIntegrationStatusAsFinished sets the RHTAP integration status condition for the Snapshot to Finished.
 func SetSnapshotIntegrationStatusAsFinished(snapshot *applicationapiv1alpha1.Snapshot, message string) {
 	condition := metav1.Condition{
-		Type:    HACBSIntegrationStatusCondition,
+		Type:    RHTAPIntegrationStatusCondition,
 		Status:  metav1.ConditionTrue,
-		Reason:  HACBSIntegrationStatusFinished,
+		Reason:  RHTAPIntegrationStatusFinished,
 		Message: message,
 	}
 	meta.SetStatusCondition(&snapshot.Status.Conditions, condition)
 }
 
-// IsSnapshotNotStarted checks if the HACBS Integration Status condition is not in progress status.
+// IsSnapshotNotStarted checks if the RHTAP Integration Status condition is not in progress status.
 func IsSnapshotNotStarted(snapshot *applicationapiv1alpha1.Snapshot) bool {
-	condition := meta.FindStatusCondition(snapshot.Status.Conditions, HACBSIntegrationStatusCondition)
-	if condition == nil || condition.Reason != HACBSIntegrationStatusInProgress {
+	condition := meta.FindStatusCondition(snapshot.Status.Conditions, RHTAPIntegrationStatusCondition)
+	if condition == nil || condition.Reason != RHTAPIntegrationStatusInProgress {
```

Comment:
may this incompatible change cause issues to ongoing requests because name of key changed?

```diff
-	// HACBSIntegrationStatusCondition is the condition for marking the HACBS integration status of the Snapshot.
-	HACBSIntegrationStatusCondition = "HACBSIntegrationStatus"
+	// RHTAPIntegrationStatusCondition is the condition for marking the RHTAP integration status of the Snapshot.
+	RHTAPIntegrationStatusCondition = "RHTAPIntegrationStatus
```

Reply:
I think we'll need to support both statuses, the similar way to how we support the legacy `HACBS_TEST_OUTPUT` name.